<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botonProductoA">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Producto A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonProductoB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Producto B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonProductoC">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Producto C</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHablar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelGraficoC">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="source1.qrc"/>
 </resources>
 <connections/>
</ui>
